docs/reference.rst

```diff
@@ -770,6 +770,20 @@ and return a value.
     >>> LogFactory(timestamp=now - timedelta(days=1))
     <Log: log at 2016-02-11 17:02:34>
 
+:class:`LazyFunction` is also useful for assigning copies of mutable objects
+(like lists) to an objects' property. Example:
+
+.. code-block:: python
+
+    DEFAULT_TEAM = ['Player1', 'Player2']
+
+    class TeamFactory(factory.Factory):
+        class Meta:
+            model = models.Team
+
+        teammates = factory.LazyFunction(lambda: list(DEFAULT_TEAM))
```

Comment:
I'm afraid that despite being a maintainer, it's been a while since I played with this...  but is the `lambda` actually needed? Since it's a `LazyFunction`, it isn't evaluated until called... am I missing something?

Reply:
I consulted the docs on this one. According to the section about [LazyFunction](http://factoryboy.readthedocs.io/en/latest/reference.html#lazyfunction): "... that method should not take any argument ...". Given that the call to "list" being done needs an argument (DEFAULT_TEAM) I think this is the correct way to go. Also, rbarrois suggested [something similar](https://github.com/FactoryBoy/factory_boy/issues/342#issuecomment-282502869) to solve issue #342.